Libplanet/Blockchain/BlockChain.cs

```diff
@@ -210,6 +249,34 @@ public Block<T> Tip
             }
         }
 
+        /// <summary>
+        /// Mine the genesis block of the blockchain.
+        /// </summary>
+        /// <param name="actions">List of actions will be included in the genesis block.
+        /// If it's null, it will be replaced with <see cref="ImmutableArray{T}.Empty"/>
+        /// as default.</param>
+        /// <param name="privateKey">A private key to sign the transaction in the genesis block.
```

Comment:
IMHO it would be better if there is a method which takes transactions instead of actions, and this method becomes a facade on that.

```suggestion
        /// <param name="privateKey">A private key to sign transactions in the genesis block.
```